torch/csrc/jit/codegen/cuda/python_frontend/fusion_record.h

```diff
@@ -0,0 +1,256 @@
+#pragma once
+#include <c10/util/complex.h>
+#include <torch/csrc/jit/codegen/cuda/arith.h>
+#include <torch/csrc/jit/codegen/cuda/ops/normalization.h>
+#include <torch/csrc/jit/codegen/cuda/python_frontend/fusion_definition.h>
+
+namespace nvfuser {
+struct RecordFunctor {
+  RecordFunctor(std::vector<size_t> _args, std::vector<size_t> _outputs)
+      : args(std::move(_args)), outputs(std::move(_outputs)) {}
+  virtual void operator()(FusionDefinition& fd) = 0;
+
+  std::vector<size_t> args;
```

Comment:
IIUC, these are trying to create a serialized view of all the fusion ops/vals, and the info kept here are referring to the indices on the serial container in `FusionDefinition`. Is that right? 

Would be helpful to comment here since the element type is size_t.

Reply:
Full comments were added.